src/components/periodic-report/periodic-report.repository.ts

```diff
@@ -156,21 +155,32 @@ export class PeriodicReportRepository extends DtoRepository(IPeriodicReport) {
   }
 
   async getLatestReportSubmitted(parentId: ID, type: ReportType) {
-    return await this.db
+    const res = await this.db
       .query()
       .match([
         node('', 'BaseNode', { id: parentId }),
         relation('out', '', 'report', { active: true }),
-        node('pr', `${type}Report`),
+        node('node', `${type}Report`),
         relation('out', '', 'start', { active: true }),
         node('sn', 'Property'),
       ])
-      .raw(`where (pr)-->(:FileNode)<--(:FileVersion)`)
-      .return('pr.id as id')
+      .raw(`where (node)-->(:FileNode)<--(:FileVersion)`)
+      .with('node, sn')
```

Comment:
Not a big deal, but I don't think the with is necessary since we `limit 1` below.